.github/workflows/test.yml

```
@@ -0,0 +1,124 @@
+        with:
+          version: ${{ matrix.emacs-version }}
+
+      - name: Install dependencies
+        run: |
+          emacs --version
+          # Create minimal package initialization
+          mkdir -p ~/.emacs.d
+          echo "(require 'package)" > ~/.emacs.d/init.el
+          echo "(add-to-list 'package-archives '(\"melpa\" . \"https://melpa.org/packages/\") t)" >> ~/.emacs.d/init.el
+          echo "(package-initialize)" >> ~/.emacs.d/init.el
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)